doc/src/manual/noteworthy-differences.md

```diff
@@ -351,6 +351,101 @@ For users coming to Julia from R, these are some noteworthy differences:
     it's more general than that since methods are dispatched on every argument type, not only `this`,
     using the most-specific-declaration rule).
 
+### Julia &hArr; C/C++: Namespaces
+  * C/C++ `namespace`s correspond roughly to Julia `module`s.
+  * There are no private functions/variables/modules/... in Julia.  Everthing is accessible
+    through fully qualified paths (or relative paths, if desired).
+  * `using MyNamespace::myfun` (C++) corresponds roughly to `import MyModule: myfun` (Julia).
+  * `using namespace MyNamespace` (C++) corresponds roughly to `using MyModule` (Julia)
+    * In Julia, only `export`ed symbols are made available to the calling module.
+    * In C++, only elements found in the included (public) header files are made available.
+  * Caveat: `import`/`using` keywords (Julia) also *load* modules (see below).
+  * Caveat: `import`/`using` (Julia) works only at the global scope level (`module`s)
+    * In C++, `using namespace X` works within arbitrary scopes (ex: function scope).
+
+### Julia &hArr; C/C++: Module loading
+  * When you think of a C/C++ "**library**", you are likely looking for a Julia "**package**".
+    * Caveat: C/C++ libraries often house multiple "software modules" whereas Julia
+      "packages" typically house one.
+    * Reminder: Julia `module`s are global scopes (not necessarily "software modules").
+  * **Instead of build/`make` scripts**, Julia uses "Project Environments" (sometimes called
+    either "Project" or "Environment").
+    * Build scripts are only needed for more complex applications
+      (like those needing to compile, or download C/C++ executables :) ).
+    * C/C++ code typically target more conventional applications, whereas Julia
+      "Project Environments" provide a set of packages to experiment with particular problem
+      spaces. Julia users typically use problem-specific "scripts" for this type of experimentation.
```

Comment:
seems a normative opinion, and does not seem to say anything prescriptive about either language
```suggestion
```

Reply:
Agreed. This is probably misleading:
- I was trying to address how C++ doesn't focus on interactive (REPL-based) sessions.
  (Unless using a special distro like CERN/Root https://root.cern/)
- Most C++ toolsets/examples help you build standalone applications.
- The C++ "project environment" is ***implicitly*** defined by the build scripts
  (including source/libraries being compiled/linked).

On the Julia side, I've noticed creating standalone applications can be tough for new users:
- Running/programming in Julia is just a different mindset.
- Interactive sessions feels more like Matlab flow - alien to some coming from C++/VisualBasic/etc.
- Can't just double-click on a .jl file in Windows to launch a Julia script.
  (No file associations are created on the Julia-Windows install)

It might be nice to be able to get Windows distributions set up such that the following file would just work by double-clicking:
```julia
#!JLScript -iq --startup-file=no --project=@Global_ProjectNameHere

using Plots
#[Do computations here]
#[Generate plothere]

```

Anyhow. I'm ok with dropping this. This might not be the right place to address this anyhow.